models/repo/repo.go

```diff
@@ -821,3 +821,11 @@ func FixNullArchivedRepository(ctx context.Context) (int64, error) {
 		IsArchived: false,
 	})
 }
+
+// UpdateRepositoryOwnerName updates the owner name of all repositories owned by the user
+func UpdateRepositoryOwnerName(ctx context.Context, oldUserName, newUserName string) error {
+	if _, err := db.GetEngine(ctx).Exec("UPDATE `repository` SET owner_name=? WHERE owner_name=?", newUserName, oldUserName); err != nil {
+		return fmt.Errorf("change repo owner name: %w", err)
+	}
+	return nil
+}
```

Comment:
What is the difference in `UpdateRepositoryOwnerName` and `UpdateRepositoryOwnerNames` ?
https://github.com/go-gitea/gitea/blob/cc8874864ca44ec013e132ebcb80b40878010d12/models/repo/update.go#L18-L33

Reply:
One is for updating one repository, another is for updating all owner's repositories.